dgraphtest/dgraph.go

```diff
@@ -241,3 +248,50 @@ func publicPort(dcli *docker.Client, dc dnode, privatePort string) (string, erro
 
 	return "", fmt.Errorf("no mapping found for private port [%v] for container [%v]", privatePort, dc.cname())
 }
+
+func mountDBinary(c *LocalCluster) (mount.Mount, error) {
+	if c.conf.version != "local" {
```

Comment:
I think you should handle the local case first and return. It will reduce indentation in the code